.semaphore/semaphore.yml

```diff
@@ -67,12 +79,9 @@ blocks:
         commands:
           - checkout
           - cache restore gems-$SEMAPHORE_GIT_BRANCH-$(checksum Gemfile.lock),gems-$SEMAPHORE_GIT_BRANCH-,gems-master-
-          # Start Postgres database service.
           # See https://docs.semaphoreci.com/article/54-toolbox-reference#sem-service
-          - sem-service start postgres
-          - sem-version ruby 2.6.0
           - bundle install --deployment --path vendor/bundle
-          - bundle exec rake db:setup
+          - bundle exec rake db:migrate db:setup
```

Comment:
I had to add `db:migrate` but I don't fully understand why.

With only `db:setup`:

<img width="633" alt="Screen Shot 2019-05-14 at 1 56 55 PM" src="https://user-images.githubusercontent.com/1779493/57696057-2ca54280-7650-11e9-9ee1-8ba516fb080e.png">

My hypothesis:

I've changed the `RAILS_ENV` to `test`. Previously this was not set and the default `development` environment was assumed.